integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -144,6 +144,7 @@ object BloopDefaults {
       Option(System.getProperty("bloop.export-jar-classifiers"))
         .orElse(Option(System.getenv("BLOOP_EXPORT_JAR_CLASSIFIERS")))
         .map(_.split(",").toSet)
+        .orElse(Some(Set("sources", "javadoc")))
```

Comment:
@tgodzik `sources` classifier is useful for metals, but is `javadoc` useful? I do wonder if this isn't too much, wdyt?

Reply:
It might be useful in some potential cases, though it's unused currently in Metal